src/main/features/dashboard/views/macro/claimStatus/claimFullAdmissionForDefendant.njk

```diff
@@ -5,14 +5,25 @@
 
 {% macro claimFullAdmissionForDefendantDashboard(claim) %}
 
-  {% if claim.response.paymentIntention.paymentOption === domain.PaymentOption.IMMEDIATELY %}
-    {{ t('You’ve admitted all of the claim and said you’ll pay the full amount immediately.') }}
-  {% elseif claim.response.paymentIntention.paymentOption === domain.PaymentOption.BY_SPECIFIED_DATE %}
-    {{ t('You’ve admitted all of the claim and offered to pay the full amount by {{ paymentDate }}.',
-      { paymentDate: claim.response.paymentIntention.paymentDate | date }) }}
-  {% elseif  claim.response.paymentIntention.paymentOption === domain.PaymentOption.INSTALMENTS %}
-    {{ t('You’ve admitted all of the claim and offered to pay the full amount in instalments.') }}
+  {% if claim.claimData.defendant.isBusiness() %}
```

Comment:
Please check with content guys as this content is almost identical. Ask them to update the prototype so both versions are the same then remove the if statement for isBusiness.

Reply:
the only one of these three that is different is the by set date. please move the if in side there rather than duplicating the other content.